test/test_model.py

```diff
@@ -3370,16 +3398,16 @@ def test_grant(self):
         meta = ops.CharmMeta()
         cache = ops.model._ModelCache(meta, backend)
         unit = ops.Unit('test', meta, backend, cache)
-        rel123 = ops.Relation('test', 123, True, unit, backend, cache)
-        rel234 = ops.Relation('test', 234, True, unit, backend, cache)
+        rel123 = ops.Relation('test', 123, True, unit, backend, cache, True)
```

Comment:
Similar in this file, I'd recommend `active=True` (or, probably better, just make the default True).